lib/Core/PTree.cpp

```diff
@@ -9,14 +9,35 @@
 
 #include "PTree.h"
 
+#include "klee/ExecutionState.h"
 #include "klee/Expr/Expr.h"
 #include "klee/Expr/ExprPPrinter.h"
+#include "klee/Internal/Module/KInstruction.h"
+#include "klee/Internal/Support/FileHandling.h"
+#include "klee/Internal/Support/ErrorHandling.h"
+#include "StatsTracker.h"
 
 #include <vector>
 
 using namespace klee;
 
-PTree::PTree(const data_type &root) : root(new Node(nullptr, root)) {}
+PTree::PTree(const data_type &root)
+    : root(new Node(nullptr, root)) {
+
+  if (StatsTracker::dumpProcessTree()) {
+    std::string error;
+    csvOutputFile = klee_open_output_file("klee-last/tree.csv", error);
```

Comment:
```suggestion
    csvOutputFile = klee_open_output_file("tree.csv", error);
```

Reply:
Why shouldn't the csv file be in the klee-last directory?

Reply:
`klee_open_output_file` should put it there. Note that `klee-last` is not safe, becasue there can be two instances of klee running in the same working directory and the klee-last would only point to klee-out- of one of them.